bot/events/rssFeeds.js

```diff
@@ -0,0 +1,43 @@
+const log = require('another-logger');
+const {EventListener} = require('yuuko');
+
+const Parser = require('rss-parser');
+const rssParser = new Parser();
+
+module.exports = new EventListener('ready', ({client, db}) => {
+	const collection = db.collection('rssFeeds');
+
+	// This function calls itself every 60 seconds as long as the bot is running.
```

Comment:
Might want to indicate here that it's a 60 second delay *between* checks to avoid expectations that it runs at the same time every minute.

Reply:
This is probably something I should clarify in the other periodic actions too - not necessary to go into those files for this PR though